dolphinscheduler-ui/src/views/resource/udf/resource/components/upload-modal.tsx

```diff
@@ -52,12 +52,20 @@ export default defineComponent({
     const customRequest = ({ file }: any) => {
       state.uploadForm.name = file.name
       state.uploadForm.file = file.file
+      state.uploadFormRef.validate()
+    }
+
+    const removeFile = () => {
+      state.uploadForm.name = ''
+      state.uploadForm.file = ''
+      state.uploadFormRef.validate()
```

Comment:
No verification is required here.

Reply:
Agree. I'll add a commit to cancel the verification.